templates/header.html

```diff
@@ -13,6 +13,7 @@
             <li {% if current_page == "articles" %}aria-current="page"{% endif %}><a href="/articles/">Articles</a></li>
             <li {% if current_page == "donate" %}aria-current="page"{% endif %}><a href="/donate">Donate</a></li>
             <li {% if current_page == "contact" %}aria-current="page"{% endif %}><a href="/contact">Contact</a></li>
+            <li {% if current_page == "hiring" %}aria-current="page"{% endif %}><a href="/hiring">Hiring</a></li>
```

Comment:
Please leave this out for now. It can be added to the navigation bar once it's more heavily reviewed and edited by others.